README.md

```diff
@@ -1,26 +1,17 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Arrow Dan
 
 > Description the project.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+- HTML
+- CSS
 
 ## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
+**My very first project in Microverse and where I did personal Review**
```

Comment:
Almost there, just so close
![image](https://user-images.githubusercontent.com/70237052/161091786-4b5a3a53-6289-44b6-b37a-c7ffd033156c.png)
In this project description, kindly make it specific and descriptive enough to communicate.